16 Lazy Loading/package.json

```diff
@@ -27,12 +28,13 @@
     "extract-text-webpack-plugin": "^1.0.1",
     "file-loader": "^0.8.5",
     "html-webpack-plugin": "^2.9.0",
+    "rimraf": "^2.5.4",
```

Comment:
This are dev dependcies, e.g. rimraf is use to remove folders / files (since this command is different in windows and mac a good fellow made this package to allow this).